content/v2/certificates.markdown

```diff
@@ -136,3 +136,196 @@ Responds with HTTP 200, renders the certificate private key.
 ~~~json
 <%= pretty_print_fixture("/getCertificatePrivateKey/success.http") %>
 ~~~
+
+## Purchase a Let's Encrypt certificate {#letsencrypt-purchase}
+
+    POST /:account/domains/:domain/certificates/letsencrypt
+
+Purchase a [Let's Encrypt](https://dnsimple.com/letsencrypt) certificate with DNSimple.
+
+<note>
+Please note that the domain must be [delegated](https://support.dnsimple.com/articles/pointing-domain-to-dnsimple/) to DNSimple.
+</note>
+
+The certificate is **free of charge**, but some features of the certificate may require to be subscribed to an higher plan.
+
+### Certificate name
+
+The default certificate name is `www`, which is valid for the naked domain as well.
+The name of the certificate will be `www.example.com`, which is valid both for `https://example.com` and `https://www.example.com`.
+
+You can choose a **custom name** (like `api`), which is valid only for `https://api.example.com`.
+Custom name requires a subscription to Professional or a Business plan.
+
+### Alternate names
+
+A certificate can be purchased for multiple subdomains.
+We call them _alternate names_ or [Subject Alternative Name (SAN)](https://support.dnsimple.com/articles/what-is-ssl-san/).
+
+By default a certificate doesn't have alternate names.
+
+You can purchase a single certificate for both `https://docs.example.com` and `https://status.example.com`, alongside with `https://example.com`.
+Alternate names require a subscription to Professional or a Business plan.
+
+### Auto renewal
+
+By default a certificate isn't auto-renewed when in expires.
+
+If that is your case, you must [manually renew](#letsencrypt-purchase-renewal) the certificate.
+
+Otherwise you can **purchase once** and let the system to renew it automatically.
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:domain` | `string`, `integer` | The domain name or id
+
+### Example
+
+Purchase a Let's Encrypt certificate for `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          -H 'Content-Type: application/json' \
+          -X POST \
+          -d '<json>' \
+          https://api.dnsimple.com/v2/1010/domains/example.com/certificates/letsencrypt
+
+
+### Input
+
+Name | Type | Description
+-----|------|------------
+`contact_id` | `integer` | **Required**. The ID of an existing contact in your account.
+`auto_renew` | `bool` | Set to true to enable the auto-renewal of the certificate. Default: `false`.
+`name` | `string` | The certificate name. Default: `"www"`.
+`alternate_names` | `array<string>` | The certificate _alternate names_. Default: `[]`. Example: `["docs.example.com", "status.example.com"]`
+
+<note>
+The `contact_id` can be fetched via the [contacts endpoint](/v2/contacts) and will be the contact for this certificate.
```

Comment:
I think the second part of the sentence, _and will be the contact for this certificate_, can be removed.